generators/utils-github-templates/templates/plain/.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,65 @@
+## Release notes
+
+*List of addressed issues and PRs since the last release*
+
+
+## Checklists
+
+### Release preperation
+
+* [x] Create new `release-x.x.x` branch (based on `develop` branch)
+* [ ] Check selected branches (from `release-x.x.x` into `master` branch)
+* [ ] Collect changes and write [release notes](#release-notes)
+
+### Release dependencies first
+
+In case of dependent Phovea/TDP repositories follow [dependency tree](https://wiki.datavisyn.io/phovea/fundamentals/development-process#dependency-hierarchy) from the top:
+
+* [ ] Release dependent repositories if they contain changes first before proceeding here
+* [ ] Replace git dependencies in *package.json* with new version range (e.g., `"phovea_core": "~2.3.1"`)
+* [ ] Replace git dependencies in *requirements.txt* with new version range (e.g., `phovea_server>=2.3.0,<2.4`)
+* [ ] Commit and push new dependencies
+* [ ] Wait until build is successful
+* [ ] Repeat with other repositories/dependencies or proceed with next section
+
+### Update version
+
+* [ ] Check version numbers of dependencies again
+* [ ] Check if build is successful
+* [ ] Update this version number following [semver](https://semver.org)
```

Comment:
Should the user edit the _package.json_ manually? If we use `npm version major|minor|patch` it will automatically create a git tag which changes the github release process (see below)?

Reply:
I will do it by hand for now. We can try to automate it later.